### PR TITLE
Fixes #34971 - fix the validators label help

### PR DESCRIPTION
**Pull request by ares**
before:
v_type_before
after:
v_type_after

you need to have a foreman_ansible or foreman_puppet installed, however the code responsible for this part is in the core

### CODE REVIEW

**Comment by ares** [comment]
[test foreman]

File: app/helpers/lookup_keys_helper.rb [hunk 1]
```diff
@@ -36,10 +36,10 @@ def validator_type_selector(f)
     selectable_f f, :validator_type, options_for_select(LookupKey::VALIDATOR_TYPES.map { |e| [_(e), e] }, f.object.validator_type), {:include_blank => _("None")},
       { :disabled => (f.object.puppet? && !f.object.override), :size => "col-md-8", :class => "without_select2",
         :onchange => 'validatorTypeSelected(this)',
-        :label_help => _("<dl>" +
-          "<dt>List</dt> <dd>A list of the allowed values, specified in the Validator rule field.</dd>" +
-          "<dt>Regexp</dt> <dd>Validates the input with the regular expression in the Validator rule field.</dd>" +
-          "</dl>").html_safe,
+        :label_help => _(
+          "<b>List</b> A list of the allowed values separated by comma, specified in the Validator rule field. The default value and all values must then be one of the specified value.<br/>" +
```
Comment (MariaAga) [inline app/helpers/lookup_keys_helper.rb]: This uses the word value in a lot of contexts so I think each `value` should be specified to:
allowed value | default value |  input value 
Maybe change `specified value` to `specified allowed value` and `all values` to `all input values`?

Comment (ares) [reply app/helpers/lookup_keys_helper.rb]: good catch, I updated this and also the regexp part accordingly

**Comment by MariaAga** [comment]
rubocop looks sad

**Comment by ares** [comment]
let's make it happy then :policeman: :rainbow: :smile:

**Review by MariaAga — APPROVED** [review]
LGTM, Thanks!

**Pull request merged**